packages/editor/src/components/post-featured-image/index.js

```diff
@@ -46,30 +46,32 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 	return (
 		<PostFeaturedImageCheck>
 			<div className="editor-post-featured-image">
-				{ !! featuredImageId &&
-					<MediaUploadCheck fallback={ instructions }>
-						<MediaUpload
-							title={ postLabel.featured_image || DEFAULT_FEATURE_IMAGE_LABEL }
-							onSelect={ onUpdateImage }
-							allowedTypes={ ALLOWED_MEDIA_TYPES }
-							modalClass="editor-post-featured-image__media-modal"
-							render={ ( { open } ) => (
-								<Button className="editor-post-featured-image__preview" onClick={ open } aria-label={ __( 'Edit or update the image' ) }>
-									{ media &&
-										<ResponsiveWrapper
-											naturalWidth={ mediaWidth }
-											naturalHeight={ mediaHeight }
-										>
-											<img src={ mediaSourceUrl } alt="" />
-										</ResponsiveWrapper>
-									}
-									{ ! media && <Spinner /> }
-								</Button>
-							) }
-							value={ featuredImageId }
-						/>
-					</MediaUploadCheck>
-				}
+				<MediaUploadCheck fallback={ instructions }>
+					<MediaUpload
+						title={ postLabel.featured_image || DEFAULT_FEATURE_IMAGE_LABEL }
+						onSelect={ onUpdateImage }
+						allowedTypes={ ALLOWED_MEDIA_TYPES }
+						modalClass={ ! featuredImageId ? 'editor-post-featured-image__media-modal' : 'editor-post-featured-image__media-modal' }
```

Comment:
Both classes are the same -- I think we can just use modalClass="editor-post-featured-image__media-modal".